src/panels/ConfigPanel/ConfigPanel.tsx

```diff
@@ -215,29 +219,56 @@ export default function ConfigPanel(): JSX.Element {
   function generateConfirmationDialog(): JSX.Element | null {
     if (confirmType && appName) {
       const changedConfigList = generateChangedKeyValues(config);
-
       if (confirmType === "apply") {
-        return SaveConfirmation(
-          appName,
-          changedConfigList,
-          () => {
-            setConfirmType(null);
-            // Clear the form errors if there were any from a previous submit.
-            setFormErrors(null);
-            _submitToJuju();
-          },
-          () => setConfirmType(null)
+        return (
+          // Render the submit confirmation modal.
+          <ConfirmationModal
+            className="p-confirmation-modal"
```

Comment:
I left the `className` to be `p-confirmation-modal` here, as I thought that by doing so, the `className` would be consistent with the other `ConfirmationModel`s used in `ActionsPanel` and `CharmActionsPanel`. Please let me know if adding an extra class here or removing the current class and adding another one would make more sense.

Reply:
Just read the suggestions and I agree with them. Thank you for the review!